root/static/scripts/guess-case/MB/GuessCase/Handler/Track.js

```diff
@@ -91,10 +91,10 @@ MB.GuessCase.Handler.Track = function (gc) {
      * - Handles FeaturingArtistStyle
      */
     self.doWord = function () {
-        if (self.doIgnoreWords()) {
-        } else if (self.doFeaturingArtistStyle()) {
-        } else if (gc.mode.doWord()) {
-        } else {
+        if (self.doIgnoreWords());
+        else if (self.doFeaturingArtistStyle());
+        else if (gc.mode.doWord());
+        else {
```

Comment:
Here you can combine all three conditions into the first `if`, or assign the result to a `handled` variable like above and do `if (!handled)` for the rest of the matching below.